### PR TITLE
feat: clarify option selection mechanism

### LINKED ISSUE #1121: 📝 Documentation: Clarify how to operate the Clack prompt
### Bug Report Checklist

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

image displaying a screenshot of the terminal with a message I am unclear on

As you can probably gather, I didn't work out how and ended up with everything to my chagrin!

### Additional Info

n/a

### PR DESCRIPTION
**Pull request by johnnyreilly**
## PR Checklist

- [x] Addresses an existing open issue: fixes #1121
- [x] That issue was marked as [`status: accepting prs`](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)
- [x] Steps in [CONTRIBUTING.md](https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/.github/CONTRIBUTING.md) were taken

## Overview

This PR adds the text `You can press ↑ and ↓ on the keyboard to change the selected item, then space to select.` to the creation mechanism.

:sunflower:

### CODE REVIEW

**Comment by johnnyreilly** [comment]
I see the code coverage message above, but I don't think this changes code coverage?

File: src/shared/options/augmentOptionsWithExcludes.ts [hunk 1]
```diff
@@ -213,7 +213,7 @@ export async function augmentOptionsWithExcludes(
 					await prompts.multiselect({
 						initialValues: exclusionKeys,
 						message:
-							"Select the tooling portions you'd like to remove. All are enabled by default.",
+							"Select the tooling portions you'd like to remove. All are enabled by default. You can press ↑ and ↓ on the keyboard to change the selected item, then space to select.",
```
Comment (JoshuaKGoldberg) [inline src/shared/options/augmentOptionsWithExcludes.ts:216]: Just nitpicking to reduce the amount of text. Thoughts?

```suggestion
							"Select the tooling portions you'd like to remove. All are enabled by default. Press ↑ or ↓ to change the selected item, then space to select.",
```

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM, thanks! What do you think about this small text adjustment?

File: src/shared/options/augmentOptionsWithExcludes.ts (hunk 1 above)
Comment (johnnyreilly) [reply src/shared/options/augmentOptionsWithExcludes.ts]: Yeah less is more!

**Review by JoshuaKGoldberg — APPROVED** [review]
Swell, thanks!

**Pull request merged**